spec.md

```diff
@@ -54,7 +54,7 @@ _Fields_
 
 ### Data
 
-A `Data` object represents a static value that is assigned to a name.
+A `Data` object represents a value that is assigned to a name or an expression.
```

Comment:
This describes `(name or expression) = value` which is not right. Did you want to express `name = (value or expression)`? Then I don't know what is the meaningful difference between a value and an expression in this context.

Reply:
Actually, it's  `value = (name or expression)` that I wanted to express. English is not my mother longue.

Reply:
I see. How about this?

> A `Data` object represents an expression assigned to a name (e.g., a variable or constant).

In Python this already includes aliases because on the right hand side of the operator, the name is also just an expression.